includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1544,7 +1544,7 @@ public function get_wizard_in_cart_payment_gateways() {
 	 *
 	 * @return array
 	 */
-	protected function get_wizard_manual_payment_gateways() {
+	public function get_wizard_manual_payment_gateways() {
```

Comment:
I imagine this is okay since `get_wizard_in_cart_payment_gateways()` is also public.